tests/validation/tests/v3_api/test_alert.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+import requests
+
+from .common import *  # NOQA
+from .alert_common import MockReceiveAlert
+
+dingtalk_config = {
+    "type"  : "/v3/schemas/dingtalkConfig",
+    "url"   : "http://127.0.0.1:4050/dingtalk/",
```

Comment:
I wonder if the loopback address is accessible in the validation test environment. If that's the case, we can do it in integration tests for testing the send action. I will leave it to QA for verification